src/room/GroupCallView.tsx

```diff
@@ -288,17 +291,23 @@ export const GroupCallView: FC<Props> = ({
 
   const { t } = useTranslation();
 
-  if (roomEncrypted && !perParticipantE2EE && !e2eeSharedKey) {
+  if (e2eeSystem.kind === E2eeType.NONE && !widget) {
+    // the url wants encryption, but we don't have a encryption system. (e.g. when joining a call without password)
```

Comment:
How do we know this isn't supposed to be an unencrypted call? Does this change break unencrypted calls?

Reply:
I'm not sure I get everything you've said, but… all I'm worried about is whether we're breaking existing call links. So if a link like https://call.element.dev/room/#/robin-unencrypted?roomId=!FkGOYsKvcjIQJsNpiu:call-unstable.ems.host, which (currently) leads to a unencrypted SPA call, will continue to work, then we're on the same page

Reply:
from the initial error message i figured that unenecrypted calls are never supported...
So yea the link you shared would be broken.
A shared secret room is also an unencrypted room.
So i though if the room is not encrypted and we dont have a key from the url or local storage this already implies we reach a  failiour mode (since all calls have to be encrypted)
Otherwise the new hook basically covers all cases
encrypted room -> per sender keys
unenecrypted room 
   - with password in url -> shared secret
   - with password in local storage -> shared secret
   - no password -> unenecrypted call
   
So I think we can entirely skip the error screen.
This makes it also possible to open a guest link to an unencrypted room in the spa.

Reply:
Updated it to allow it intentional unencrypted calls.